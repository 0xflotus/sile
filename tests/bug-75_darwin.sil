\begin[papersize=a5]{document}
\script[src=packages/bidi]
\thisframeLTR%
\font[family=Kohinoor Devanagari,size=10pt,language=hi]
यह दाग़ दाग़ उजाला
\end{document}
