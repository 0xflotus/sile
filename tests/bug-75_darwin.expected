<tests/bug-75_darwin.sil>
Set paper size 	419.5275636	595.275597
Begin page
Mx 	40.97638
My 	36.10378
Set font 	Kohinoor Devanagari;10;200;normal;normal;;LTR
T	78 86 3 70 21 89 3 70 21 89 3 8 60 21 80 21 3	(यह दाग़ दाग़ उजाला )
[1] 
! Font 'Gentium' not available, falling back to 'Times'
Mx 	207.26378
My 	554.36253
Set font 	Gentium;10;200;normal;normal;;LTR
T	20	(1)
End page
Finish

